main.nf

```diff
@@ -434,8 +435,9 @@ process search_engine_msgf {
                      -threads ${task.cpus} \\
                      -database ${database} \\
                      -matches_per_spec ${params.num_hits} \\
-                     -fixed_modifications "${fixed}" \\
-                     -variable_modifications "${variable}" \\
+                     -variable_modifications ${params.variable_mods.tokenize(',').collect { "'${it}'" }.join(" ") } \\
+                     -fixed_modifications ${params.fixed_mods.tokenize(',').collect { "'${it}'"}.join(" ")} \\
+                     -enzyme ${params.enzyme} \\
```

Comment:
enzyme parameter is also missing for the PercolatorAdapter, but seems to have a slightly different name space

Reply:
it is not the namespace. the settings now need to be joined from the settings channels defined in the beginning. since they can come from the params OR from the sdrf file.

Reply:
ok will have a look at this